content/messaging/customer-chat-api/v3.2/index.mdx

```diff
@@ -2791,6 +2806,41 @@ Here's what you need to know about **webhooks**:
 
 You can create and manage webhooks via the [Configuration API](/management/configuration-api/#webhooks). Refer to this document to see the list of **available webhooks**, as well as **sample payloads**.
 
+# Errors
+
+<CodeResponse title={'General error format'}>
+
+```json
+{
+  "error": {
+    "type": "misdirected_request",
+    "message": "Wrong region",
+    "data": {
+      // optional
+      "region": "dal"
+    }
+  }
+}
+```
+
+</CodeResponse>
+
+#### Possible errors
+
+| Type                        | Default Message               | Notes                                                              |
+| --------------------------- | ----------------------------- | ------------------------------------------------------------------ |
+| `authentication`            | Authentication error          | An invalid or expired access token.                                |
+| `authorization`             | Authorization error           | Agent is not allowed to perform the action.                        |
+| `entity_too_large`          | Upload limit exceeded (10MB). | Client's request is too large.                                     |
+| `internal`                  | Internal server error         |                                                                    |
+| `license_expired`           | License expired               | The end of license trial or subcription.                           |
+| `license_not_found`         | License not found             | License with the specified ID doesn't exist.                       |
+| `misdirected_request`__*__  | Wrong region                  | Client's request should be performed to another region.            |
```

Comment:
what's this `__*__`? in some other file I've seen `**/***` next to `misdirected_request`